pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -91,6 +91,9 @@ type VitessShardSpec struct {
 
 	// TopologyReconciliation is inherited from the parent's VitessClusterSpec.
 	TopologyReconciliation *TopoReconcileConfig `json:"topologyReconciliation,omitempty"`
+
+	// UpdateStrategy is inherited from the parent's VitessClusterSpec.
+	UpdateStrategy *VitessClusterUpdateStrategy `json:"updateStrategy,omitempty"`
```

Comment:
We need to call `DefaultUpdateStrategy()` from `DefaultVitessShard()` now, so VitessShard controller sees the right, non-nil defaults before VitessKeyspace controller gets around to filling in the new field.

Reply:
Done!